calico/felix/felix.py

```diff
@@ -22,6 +22,8 @@
 
 # Monkey-patch before we do anything else...
 from gevent import monkey
+from calico.felix.tracking import UpdateMonitor
+
```

Comment:
Why is this import here, rather than with the other calico imports (and in the middle of monkey patching)? I suspect you want to move this lower (or comment why you don't).